### PR TITLE
Add seed dns overwrite option to mla links

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Add seed dns overwrite option to mla links

### Which issue(s) this PR fixes

Fixes https://github.com/kubermatic/kubermatic/issues/7964

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
/hold
wait for backend to be merged https://github.com/kubermatic/kubermatic/pull/8339

File: src/app/cluster/cluster-details/mla/alertmanager-config/component.ts [hunk 1]
```diff
@@ -66,7 +68,10 @@ export class AlertmanagerConfigComponent implements OnInit, OnDestroy {
     this._datacenterService
       .getDatacenter(this.cluster.spec.cloud.dc)
       .pipe(takeUntil(this._unsubscribe))
-      .subscribe(datacenter => (this._seed = datacenter.spec.seed));
+      .pipe(tap(datacenter => (this._seed = datacenter.spec.seed)))
+      .pipe(takeUntil(this._unsubscribe))
```
Comment (maciaszczykm) [inline src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:72]: `takeUntil` should be in the last pipe.

Comment (kgroschoff) [reply src/app/cluster/cluster-details/mla/alertmanager-config/component.ts]: absolutely! good catch. fixed.

**Pull request merged**